src/main/java/org/wecancoeit/reviews/Review.java

```diff
@@ -0,0 +1,19 @@
+package org.wecancoeit.reviews;
+
+public class Review {
+
+    private Long id;
+    private String product;
+    private String review;
+
+    public Long getId() {
+        return id;
+    }
+
+    public Review(long id, String product, String review) {
+        this.id=id;
+        this.product = product;
+        this.review = review;
+    }
+
```

Comment:
add getters so that you will be able to reference them in your thymeleaf templates